__tests__/app.test.js

```diff
@@ -13,27 +13,57 @@ afterAll(() => db.end());
 describe("GET /api", () => {
     test("200 responds with object containing available endpoints", () => {
         return request(app)
-        .get("/api")
-        .expect(200)
-        .expect("Content-Type", "application/json; charset=utf-8")
-        .then(( { body } ) => {
-            expect(JSON.stringify(body)).toBe(JSON.stringify(currentEndpointsTest));
-        })
-    })
-})
+            .get("/api")
+            .expect(200)
+            .expect("Content-Type", "application/json; charset=utf-8")
+            .then(({ body }) => {
+                expect(JSON.stringify(body)).toBe(
+                    JSON.stringify(currentEndpointsTest)
+                );
+            });
+    });
+});
 
 describe("GET /api/topics", () => {
     test("200 returns an array of topic objects", () => {
         return request(app)
             .get("/api/topics")
             .expect(200)
-            .then(( { body } ) => {
+            .then(({ body }) => {
                 const { topics } = body;
                 expect(topics.length).toBe(3);
                 topics.forEach((topic) => {
                     expect(typeof topic.description).toBe("string");
                     expect(typeof topic.slug).toBe("string");
-                })
-            })
-    })
-})
+                });
+            });
+    });
+});
+
+describe("GET /api/articles/:article_id", () => {
+    test("200 returns an article object with corresponding ID", () => {
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .expect("Content-Type", "application/json; charset=utf-8")
+            .then(({ body }) => {
+                const { article } = body;
+                expect(typeof article.author).toBe("string");
+                expect(article.author).toEqual("butter_bridge");
```

Comment:
We can use the typeof expect statements here, rather than having to check the value of each individual key 👍